build/ci/community/create_demo_pr.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -xel
+set +o pipefail
+
+cd $(dirname $csv_file)
+chmod 547 $(basename $csv_file) 
+declare -a operator_image_fields=(
+    ".spec.install.spec.deployments[0].spec.template.spec.containers[0].image"
+    ".metadata.annotations.containerImage"
+    ".spec.relatedImages[0].image"
+)
+for image_field in "${operator_image_fields[@]}"
+do
+    yq eval "$image_field |= env(operator_image_for_test)" $(basename $csv_file) -i
+done
+cd -
+
+echo $github_token > github_token.txt
+gh auth login --with-token < github_token.txt
+gh repo fork operator-framework/community-operators --clone community-operators-fork
+cd community-operators-fork
+git remote set-url origin https://csiblock:$github_token@github.com/csiblock/community-operators.git
+git fetch upstream
+git rebase upstream/master
+git push origin master --force
```

Comment:
could this also be a function? with a name about whatever it doesn? something like fork_and_clone?

Reply:
`update` seems pretty general to me.
would something like `sync_fork(_master_branch)?_from_upstream` be more accurate?